simul/matplotlib/stats.py

```diff
@@ -6,7 +6,8 @@
 
 # Our CSVs have a space after the comma, so we need a new 'dialect', here
 # called 'deploy'
-csv.register_dialect('deploy', delimiter=',', doublequote=False, quotechar='', lineterminator='\n', escapechar='',
+csv.register_dialect('deploy', delimiter=',', doublequote=False, quotechar='',
+                     lineterminator='\n', escapechar='',
                      quoting=csv.QUOTE_NONE, skipinitialspace=True)
 
 
```

Comment:
For all the python scripts, I guess we can make an exception out of rule of comments and stuff ;)